Firmware/printer.cfg

```diff
@@ -9,23 +9,46 @@
 # file named "firmware.bin" on an SD card and then restart the SKR
 # mini with that SD card.
 
-# See the example.cfg file for a description of available parameters.
+## Voron Design VORON 0.1 SKR Mini E3 V2 config
+
+## *** THINGS TO CHANGE/CHECK: ***
+## MCU path										[mcu] section
+## Z Motor currents								[tmc2209 stepper_z] section
+## Thermistor types								[extruder] and [heater_bed] sections - See 'sensor types' list at end of file
+## Extruder motor currents						[extruder] section
+## PID tune										[extruder] and [heater_bed] sections
+## Fine tune E steps							[extruder] section
+## For more info								check https://docs.vorondesign.com/build/startup/#v0
 
 [mcu]
-##	Obtain mcu value by "ls -l /dev/serial/by-id/" 
-#serial: /dev/serial/by-id/usb-Klipper_stm32f103xe_REPLACE_WITH_YOUR_VALUE
+#####################################################################
+# Obtain definition by "ls -l /dev/serial/by-id/"
+#####################################################################
+serial: /dev/serial/by-id/usb-Klipper_stm32f103xe_REPLACE_WITH_YOUR_VALUE
+
+[printer]
+kinematics: corexy
+max_velocity: 200
+max_accel: 2000
+max_z_velocity: 15
+max_z_accel: 45
+square_corner_velocity: 6.0
 
+#####################################################################
+#      X/Y Stepper Settings
+#####################################################################
 
 [stepper_x]
 step_pin: PB13
-dir_pin: PB12           # check https://docs.vorondesign.com/build/startup/#v0 if moving in the wrong direction
+dir_pin: PB12								# Add ! if moving opposite direction
```

Comment:
this tip needs to get users to the homing direction graphic somehow because "wrong" direction for a single motor in corexy doesnt necessarily mean adding/removing the !